<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maditor::View::InspectorView</class>
 <widget class="QWidget" name="Maditor::View::InspectorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="viewport" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>36</width>
     <height>69</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="traceWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="scrollarea" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="itemsWidget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>18</width>
         <height>10</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
